src/lib/DOMPointReadOnly.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import GLOBAL from './global';
+
+let domPointROExport = ('DOMPointReadOnly' in GLOBAL) ? DOMPointReadOnly : null;
+
+if (!domPointROExport) {
+  const PRIVATE = Symbol('@@webxr-polyfill/DOMPointReadOnly');
```

Comment:
We should non-conditionally export this symbol: `export const PRIVATE = Symbol('...')`